python/cudf/cudf/tests/test_s3.py

```diff
@@ -533,3 +533,17 @@ def test_write_chunked_parquet(s3_base, s3so):
             actual.sort_values(["b"]).reset_index(drop=True),
             cudf.concat([df1, df2]).sort_values(["b"]).reset_index(drop=True),
         )
+
+
+def test_no_s3fs_on_cudf_import():
```

Comment:
This test is useful, but adding it has rained a bit on our mini parade. We now see that the `pyarrow._s3fs` problem pops up whenever we import `pyarrow.parquet` or `pyarrow.orc`. (So, much less constrained than the "remove storage only" case we originally expected).

Reply:
Thanks for doing that investigation Rick. I see you've already adjusted the remainder of the code to take that into consideration and this test is still passing, so I'm good with things as they are now.